lib/commands/alert.js

```diff
@@ -0,0 +1,53 @@
+let commands = {}, helpers = {}, extensions = {};
+
+commands.getAlertText = async function () {
+  return await this.uiautomator2.jwproxy.command('/alert/text', 'GET', {});
+};
+
+/**
+ * @typedef {Object} AcceptAlertOptions
+ * @property {?string} buttonLabel - The name of the button to click in order to
+ *                                   accept the alert. If the name is not provided
+ *                                   then the script will try to detect the button
+ *                                   automatically.
+ */
+
+ /**
+  * @param {AcceptAlertOptions} opts
+  * @throws {InvalidElementStateError} If no matching button, that can accept the alert,
+  *                                    can be found
+  * @throws {NoAlertOpenError} If no alert is present
+  */
+commands.mobileAcceptAlert = async function (opts = {}) {
+  return await this.uiautomator2.jwproxy.command('/alert/accept', 'POST', opts);
+};
+
+commands.postAcceptAlert = async function () {
```

Comment:
do we need this at all? isn't the api shape the same and we can just proxy?

Reply:
no, because JSONWP endpoints are different

Reply:
really? this is what i see in basedriver:

```js
  '/wd/hub/session/:sessionId/alert/accept': {
    POST: {command: 'postAcceptAlert'}
  },
  '/wd/hub/session/:sessionId/alert/dismiss': {
    POST: {command: 'postDismissAlert'}
  },
```

Reply:
really

```
  // old alerts
  '/wd/hub/session/:sessionId/alert_text': {
    GET: {command: 'getAlertText'},
    POST: {command: 'setAlertText', payloadParams: {required: ['text']}}
  },
  '/wd/hub/session/:sessionId/accept_alert': {
    POST: {command: 'postAcceptAlert'}
  },
  '/wd/hub/session/:sessionId/dismiss_alert': {
    POST: {command: 'postDismissAlert'}
  },
  // https://w3c.github.io/webdriver/webdriver-spec.html#user-prompts
  '/wd/hub/session/:sessionId/alert/text': {
    GET: {command: 'getAlertText'},
    POST: {command: 'setAlertText', payloadParams: {required: ['text']}}
  },
  '/wd/hub/session/:sessionId/alert/accept': {
    POST: {command: 'postAcceptAlert'}
  },
  '/wd/hub/session/:sessionId/alert/dismiss': {
    POST: {command: 'postDismissAlert'}
  },
```